README.md

```diff
@@ -145,14 +145,17 @@ For example, it has an entry for the clouddriver config as:
 The two optional Spinnaker services this workflow currently supports are Fiat
 and Kayenta.
 
+In the following kustomization files, `?ref=${KUSTOMIZATION_BASE_REF}` is an optional but it's recommended to to make sure you could prevent some breaking changes from breaking your infrastructure.
```

Comment:
```suggestion
In the following kustomization files, we recommend including `?ref=${KUSTOMIZATION_BASE_REF}` to pin kustomization-base to a specific version.
```

Reply:
Thank you for the review! It'd make more sense 👍